core/src/plugins/kubernetes/commands/uninstall-garden-services.ts

```diff
@@ -22,25 +21,11 @@ export const uninstallGardenServices: PluginCommand = {
 
   handler: async ({ ctx, log }) => {
     const k8sCtx = <KubernetesPluginContext>ctx
-    const variables = getKubernetesSystemVariables(k8sCtx.provider.config)
 
-    const sysGarden = await getSystemGarden(k8sCtx, variables || {}, log)
-    const actions = await sysGarden.getActionRouter()
-
-    const graph = await sysGarden.getConfigGraph({ log, emit: false })
-    const deploys = graph.getDeploys()
-
-    log.info("")
-
-    const deployNames = deploys.map((s) => s.name)
-    const statuses = await actions.deleteDeploys({ graph, log, names: deployNames })
-
-    log.info("")
-
-    const environmentStatuses = await actions.provider.cleanupAll(log)
+    await ingressControllerUninstall(k8sCtx, log)
```

Comment:
```suggestion
     if (k8sCtx.provider.config.setupIngressController === "nginx") {
      await ingressControllerUninstall(k8sCtx, log)
    }
```
We should only remove it if it was installed by Garden or this specific Garden project in the first place.

Reply:
Right! Good catch, thank you!